### ISSUE TITLE
Do we need a world scene?

### ISSUE THREAD

**Issue by BraedonWooding** [OrderOfThePorcupine/ProjectPorcupine#114]
Through my work in #99 (Separation of the world to allow multiple worlds, and a nicer model to work with), I have eliminated the need for a `_World` scene completely (previously major segments of the UI existed prior to runtime).  Currently all that sits there is `MainCamera`, `EventSystem`, `Canvas`, `Background Camera`, `GameController`.

So it would be insanely easy to create the scene on the fly, since all that needs to change is MainCamera to become depth only, Canvas to have a UIRescaler component, BackgroundCamera to become depth -100 and to block everything (and black instead of the usual blue).  GameController is just a gameobject with the CursorCircle prefab set for the `GameController` component.

So basically all stuff that can be done in code without it becoming messy, there seems to be no speed difference (other than a few ms to create all the objects), so basically it comes down to whether or not we want a scene to change to or not.

**Comment by koosemose** [comment]
At least for now, aside from all other considerations, it is a convenience for development to be able to launch directly into an active game. 

However, I'll take your word that it would be easy, but what advantages would it offer? As is, it seems more logical to have it as a separate actual scene, and can be useful to continue having logical scene separations in the future if we add more new scenes.

**Comment by BraedonWooding** [comment]
I for one am more on the side of keeping it the same and not removing the scene.  This was more just to gather consensus and make sure I wasn't presuming something.

Keeping the convenience for development would still be pretty easy, you would just have a script that well does the same work as the 'CreateGame' (though most likely just call it).  The benefits would be mainly that our scene hierarchy is simpler, and this developer scene would disappear and so would all the code needed to support it when we build.

If we are going to add any more scenes in future then we should maintain this scene, however what scenes would we add?  Especially considering that changing scenes unloads all data, the only scenes I can think of would be multiplayer scene / 3D visual scene, though yes I agree that we should maintain the separations if adding new scenes later.

So I think keeping it the same and since it's pretty simple to change over, if we ever need to change it over we can later.

**Comment by BraedonWooding** [comment]
Closing since it's staying as it is, though feel free to comment if you think this change should be carried out and we'll reopen the issue.
